public/index.html

```diff
@@ -40,4 +40,9 @@
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
   </body>
+  <button 
+  type="dark-btn"
+  data-theme-toggle
+  aria-label="Change to dark theme"
```

Comment:
Suggesting to add a new PR that will handle all dark mode related changes 😊 So that every PR does its own thing. You can add a new branch, add the changes related to dark mode there and open a new PR.

Reply:
This has also been done now, I created a new branch called dark-mode-support and made a separate PR draft for it 🙌 

Unfortunately I suddenly began having issues with node_modules and the code no longer compiles, so atm I'm unable to remove the non-functional dark mode code/features from this branch/PR 😔 Hopefully I'll find a fix soon and clean this up so that everything can be merged successfully to main.